pybamm/models/base_model.py

```diff
@@ -421,30 +421,31 @@ def input_parameters(self):
             self._input_parameters = self._find_symbols(pybamm.InputParameter)
         return self._input_parameters
 
-    def print_parameter_info(self):
-        self._parameter_info = ""
+    def get_parameter_info(self):
+        parameter_info = []
         parameters = self._find_symbols(pybamm.Parameter)
         for param in parameters:
-            self._parameter_info += f"{param.name} (Parameter)\n"
+            parameter_info.append((param.name, "Parameter"))
```

Comment:
we should just store `param` here for more generality (and then print `param.name` in `print_parameter_info`)

Reply:
Sounds reasonable. In the specific application that started the issue the name would be sufficient, but I prefer the more general approach.